gorm-tools/src/main/groovy/gorm/tools/job/JobUtils.groovy

```diff
@@ -22,11 +22,15 @@ class JobUtils {
         return sourceId
     }
 
-    //static helper as its used both here and also in the SyncJobRenderer
-    static Map convertToMap(SyncJobEntity job){
+    static JsonOutput.JsonUnescaped getRowJobData(SyncJobEntity job) {
         // gets the raw json string and use the unescaped to it just dumps it to writer without any round robin conversion
         String jobData = job.dataToString()
-        JsonOutput.JsonUnescaped rawDataJson = JsonOutput.unescaped(jobData)
+        return JsonOutput.unescaped(jobData)
+    }
+
+    //static helper as its used both here and also in the SyncJobRenderer
+    static Map convertToMap(SyncJobEntity job){
+        JsonOutput.JsonUnescaped rawDataJson = getRowJobData(job)
 
         Map map = [
             id: job['id'],
```

Comment:
👀 look at original code, do we have every field? I think we are missing tests for "problem"